superlumic

```diff
@@ -112,10 +109,10 @@ if [ -d "/usr/local/superlumic/config" ]; then
 else
     if [ ! -z "$repo" ]; then
         setStatusMessage "Getting your config from your fork"
-        git clone -q $1 /usr/local/superlumic/config
+        git clone -q $repo /usr/local/superlumic/config
     else
         setStatusMessage "Getting the default config"
-        git clone -q https://github.com/superlumic/superlumic-config.git /usr/local/superlumic/config
+        git clone -q https://github.com/grozen/superlumic-config.git /usr/local/superlumic/config
```

Comment:
I was actually very much aware of the problematic nature of using my fork here, but I'm afraid it's a very chicken and egg situation. The two pull requests work together and so if I change it back things will just not work until the corresponding superlumic-config request is merged as well. I'd gladly just do whatever the owner prefers, if he cares to weigh in.